### ISSUE TITLE
scrcpy 2.0 does not work with samsung m21

### ISSUE THREAD

**Issue by user3477071** [Genymobile/scrcpy#3935]
- [ x] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [ x] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: [Windows 10]
 - scrcpy version: [2.0]
 - installation method: [download scrcpy-win64-v2.0.zip from https://github.com/Genymobile/scrcpy ]
 - device model: Samsung Galaxy M21
 - Android version: [13]

**Describe the bug**
```
$ ./adb devices
List of devices attached
RZ8R81E12JN     device
```

Running
`$ ./scrcpy`
gives this 'Application Error' window
```
    The application was unable to start correctly(0xc0000022)
    Click OK to close the application
```

**Comment by rom1v** [comment]
Maybe your system is 32-bit and you downloaded the 64-bit release?

**Comment by user3477071** [comment]
No, my system type is: 64-bit Operating System, x64-based processor

**Comment by rom1v** [comment]
You are one windows but you're using a "Linux" syntax (`./scrcpy`). From where do you run the program?

If you run it from a `cmd` (`double-click on `open_a_terminal_here.bat`), and type scrcpy, what happens?

Otherwise, there are many results on Google about this error (0xc0000022) on Windows, it is not related to scrcpy.
For example (random link): https://answers.microsoft.com/en-us/windows/forum/all/the-application-was-unable-to-start-correctly/e06ee08a-26c5-447a-80bd-ed339488d0f3

**Comment by user3477071** [comment]
> You are one windows but you're using a "Linux" syntax (./scrcpy). From where do you run the program?
This invocation was from a cygwin bash shell running in windows 10

> If you run it from a cmd (double-click on open_a_terminal_here.bat), and type scrcpy, what happens?
When run from cmd, I still get the same error

**Comment by user3477071** [comment]
> For example (random link): https://answers.microsoft.com/en-us/windows/forum/all/the-application-was-unable-to-start-correctly/e06ee08a-26c5-447a-80bd-ed339488d0f3

Tried workaround mentioned in above link and still get the same error
